### PR TITLE
MH-Readme fix issue #51

### PR DESCRIPTION
**Pull request by bleehu**
Fix Issue #51 
updated info in readme to talk about config generator and clear up some display mistakes. Also added some detail on startup
	modified:   README.md

Please answer with specific corrections, additions or a shipit!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
-$python start.py
+`$python start.py`
 then in a web browser, go to localhost:5000/levelup
 
 For LAN, use:
-$python start.py -i local.ip.goes.here
-to find your local ip address on linux, use ifconfig. 
-to find your local ip address on windows, use ipconfig in bash
+`$python start.py -i local.ip.goes.here`
+to find your local ip address on linux, use ifconfig in a terminal.
```
Comment (Turtlelord26) [inline README.md]: Confirm "ifconfig" as opposed to "ipconfig"?

File: README.md [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
 
 For WWW use:
-CXDocs is being tested on an AWS instance behind an NGinX SSL proxy. There are still some improvements that should be made in order to keep it safe. 
+CXDocs is being tested on an AWS instance behind an NGinX SSL proxy. There are still some improvements that should be made in order to keep it safe. Feel free to contact bleehu@uw.edu for information about how to stand up your own public instance of CX Docs.
+
+
+##Setting up configs
+Compound X comes with a handy little script for generating appropriate config files that help it do things like save pictures for the monster bestiary, etc. Generating a config is as easy as running 
+`$python generate_config.py`
+That should
```
Comment (Turtlelord26) [inline README.md]: That should what?

File: README.md (hunk 1 above)
Comment (bleehu) [reply README.md]: confirmed.

File: README.md (hunk 2 above)
Comment (bleehu) [reply README.md]: um... what if I focus on one thing at a time and finish the documentation before I ask for a review... *facepalm*

**Pull request merged**